keyring-util/src/main.rs

```diff
@@ -0,0 +1,259 @@
+use std::collections::HashMap;
+use std::fmt;
+use std::io::Write;
+use std::process::{ExitCode, Termination};
+
+use clap::parser::ValuesRef;
+use clap::{arg, value_parser, Command};
+use oo7::Keyring;
```

Comment:
it should use the dbus specific backend as the tool is not supposed as is to work with sandboxed apps